### PR TITLE
Version 0.17.1

### PR DESCRIPTION
**Pull request by Kludex**
GitHub release draft: https://github.com/encode/uvicorn/releases
Changes: https://github.com/encode/uvicorn/compare/0.17.0.post1...master

### CODE REVIEW

**Review by lovelydinosaur — APPROVED** [review]

**Comment by Kludex** [comment]
Thanks @tomchristie ! :)

**Comment by lovelydinosaur** [comment]
Great.

The deploy is waiting on approval for anyone on @encode/operations - https://github.com/encode/uvicorn/actions/runs/1760983069

@Kludex you wanna hit the go button?

**Comment by Kludex** [comment]
Hmmm... I completely forgot about that. :thinking: 

Done.

**Pull request merged**
